Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_006.GitHubLegacyPatValidator.cs

```diff
@@ -93,6 +94,7 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                     name = $" ({name})";
                 }
 
+                asset = $"{id}{name}";
```

Comment:
>asset [](http://example.com/codeflow?start=16&length=5)

another good change! we report '@michaelcfanning (Michael C. Fanning)' as the relevant tested thing now when this rule raises an exception.